src/Helpers/ActionsHelpers/ActionHelper.cs

```diff
@@ -61,5 +64,16 @@ private string RecursiveGetAnswerValue(string targetMapping, Answers answer)
             var subObject = new Answers { Response = (dynamic)answer.Response[splitTargets[1]] };
             return RecursiveGetAnswerValue(targetMapping.Replace($"{splitTargets[0]}.", string.Empty), subObject);
         }
+
+        public ValidateEntity GenerateDoumentUploadUrl(string baseUrl, FormAnswers formAnswers)
+        {
+            var matches = TagRegex.Matches(baseUrl);
+            var newUrl = matches.Aggregate(baseUrl, (current, match) => Replace(match, current, formAnswers));
+            return new ValidateEntity
+            {
+                Url = newUrl,
+                IsPost = false
+            };
```

Comment:
can this be remove, it is the same method as GenerateUrl(), the enity model on GenerateUrl can be renamed to a more generic entity.